### PR TITLE
LPS-150594 [CP2.0] Add Setup Analytics Cloud Page for Product Activation tests

### PR DESCRIPTION
**Pull request by luanakarla**

### CODE REVIEW

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:test:osb-site-initializer-customer-portal

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2SetupAC.testcase [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+
+		task ("And I go through onboarding") {
+			CP2Utils.onboardingFlow();
+		}
+
+		task ("Then I click on the AC product") {
+			CP2NavigationMenu.goToProductActivation();
+
+			CP2NavigationMenu.clickOnProductButton(product = "${testProduct}");
+		}
```
Comment (barrosdennys) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2SetupAC.testcase:82]: @luanakarla it looks like this block of code is used on all 4 tests written here. Would you mind to add it into the setUp block?

Comment (luanakarla) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2SetupAC.testcase:82]: yea, I'm adding it now. Thanks!

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:test:osb-site-initializer-customer-portal

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:test:osb-site-initializer-customer-portal

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:test:osb-site-initializer-customer-portal

**Comment by luanakarla** [comment]
ci:close

**Pull request closed without merging**